source/monitoring_audience/analyzing_data_usage.rst

```diff
@@ -1,90 +1,100 @@
 Analyzing data usage
 ====================
 
-Go to the *Analytics > Data* page.
+The Data dashboard displays statistics to analyze the data published on the portal. It is available in the Analytics > Data section of the back office.
 
-There are seven indicators :
+This dashboard is separated into 4 sections:
 
-1. **Dataset activity**: Download and API calls of most used datasets. Please note that there are two different scales for the downloads count and the number of API calls on each side of the chart.
+- Totals: figures about the datasets published in the portal and their usage through that portal
+- Popularity: statistics and charts to compute and analyze the popularity of the datasets published in the portal
+- Top 5: lists of most downloaded, used and reused published datasets
+- Theme Distribution: tree chart to analyze most and least used themes
 
-.. ifconfig:: language == 'en'
+.. admonition:: Datasets taken into account for the Data dashboard
+   :class: note
 
-    .. image:: usage__data-usage-1-en.jpg
-        :alt: Data Usages Popularity Indicator
+   In this dashboard, only published datasets are taken into account. Unpublished datasets are ignored from all figures, charts and other computed analytics displayed.
 
-.. ifconfig:: language == 'fr'
+   It is however possible, by ticking the boxes at the top of the dashboard, to choose to build the dashboard only with the private datasets (those in restricted access, no matter the restriction) or only with the public ones (those accessible by everyone).
```

Comment:
I think in the version that will be released, it will be radio buttons and not boxes anymore 😬 
![screen shot 2018-05-03 at 14 10 23](https://user-images.githubusercontent.com/5553380/39575688-bca6ece6-4edb-11e8-982a-5b8deedd7abc.png)